.gitignore

```diff
@@ -3,6 +3,6 @@
 *.iml
 .env
 .DS_Store
-venv/
+.venv/
```

Comment:
The `.venv/` entry is correctly ignoring virtual environment directories. Ensure that this is the correct directory name for your virtual environment. If your virtual environment is named differently, you should update this entry accordingly.